slacker/commands/argument_parser.py

```diff
@@ -0,0 +1,10 @@
+import argparse
+
+class DidNotExitException(BaseException):
+  pass
+
+class ArgumentParser(argparse.ArgumentParser):
```

Comment:
This is nice. I noticed that when I was making the emoji downloader and run `-h|--help` the REPL would terminate.

Reply:
Yeah at first it was weird to me why it happened, but then I remembered that the help argument would have that effect, which is normally what you want.